apps/07jquery-tic-tac-toe/script.js

```diff
@@ -1,5 +1,72 @@
 'use strict';
 
+var playerTurn = 'X';
+var boxZero = $("[data-cell='0']");
+var boxOne = $("[data-cell='1']");
+var boxTwo = $("[data-cell='2']");
+var boxThree = $("[data-cell='3']");
+var boxFour = $("[data-cell='4']");
+var boxFive= $("[data-cell='5']");
+var boxSix = $("[data-cell='6']");
+var boxSeven = $("[data-cell='7']");
+var boxEight = $("[data-cell='8']");
+
 $(document).on('ready', function() {
-  // Put app logic in here
+  $('[data-cell]').click(function() {
+    $(this).text(playerTurn);
+    checkForWin();
+    playerTurn = (playerTurn === 'X') ? 'O' : 'X';
+  })
+
+  $('#clear').click(function() {
+    $('[data-cell]').text("");
+    $('#announce-winner').text("");
+    playerTurn = 'X';
+  })
+
+
+  function horizontalWin() {
+    if((boxZero.text() === playerTurn) && (boxOne.text() === playerTurn) && (boxTwo.text() === playerTurn)) {
```

Comment:
Much easier to read and manage with the jQuery objects stored in variables.

Nice work.